packages/spore/src/codec/helper.ts

```diff
@@ -0,0 +1,27 @@
+import { Hex } from "@ckb-ccc/core";
+import { bytes, BytesLike } from "@ckb-lumos/codec";
+
+const encoder = new TextEncoder();
+const decoder = new TextDecoder();
+
+export function bytifyRawString(text: string): Uint8Array {
+  return encoder.encode(text);
+}
+
+export function bufferToRawString(
```

Comment:
and `ccc.bytesTo` here.

Reply:
ccc.bytesTo cannot accept a BytesLike from Lumos, which is used in Lumos' molecule package